pkg/kn/commands/broker/create.go

```diff
@@ -30,12 +30,15 @@ var createExample = `
   # Create a broker 'mybroker' in the current namespace
   kn broker create mybroker
 
-  # Create a broker 'mybroker' in the 'myproject' namespace
-  kn broker create mybroker --namespace myproject`
+  # Create a broker 'mybroker' in the 'myproject' namespace and with a broker class of 'Kafka'
+  kn broker create mybroker --namespace myproject --broker Kafka
```

Comment:
```suggestion
  # Create a broker 'mybroker' in the 'myproject' namespace and with a broker class of 'KafkaBroker'
  kn broker create mybroker --namespace myproject --broker KafkaBroker
```

Sorry my bad again, I think it should be KafkaBroker (so with a trailing Broker suffix.

Reply:
I am following the [KNative Kafka Broker documentation](https://knative.dev/docs/eventing/broker/kafka-broker/) and it is just Kafka as per it:

```
apiVersion: eventing.knative.dev/v1
kind: Broker
metadata:
  annotations:
    # case-sensitive
    eventing.knative.dev/broker.class: Kafka
  name: default
  namespace: default
spec:
  # Configuration specific to this broker.
  config:
    apiVersion: v1
    kind: ConfigMap
    name: kafka-broker-config
    namespace: knative-eventing
  # Optional dead letter sink, you can specify either:
  #  - deadLetterSink.ref, which is a reference to a Callable
  #  - deadLetterSink.uri, which is an absolute URI to a Callable (It can potentially be out of the Kubernetes cluster)
  delivery:
    deadLetterSink:
      ref:
        apiVersion: serving.knative.dev/v1
        kind: Service
        name: dlq-service
```

Reply:
Ok, I was referring to https://knative.dev/docs/eventing/broker/example-mtbroker/ where it used as `KafkaBroker`. @matzew which is correct now ?

Reply:
I think it's just `Kafka` because I tried creating 2 brokers with the different classes and the Kafka controller only picked up when the annotation was set to `Kafka` and not when it was set to `KafkaBroker` but we can confirm this once before we decide to push.

Reply:
Let me open an issue over there at the docs repo, to find out and get confirmation from the experts ;-)

Reply:
I just asked over there to clarify this https://github.com/knative/docs/pull/4090 (would really prefer "Kafka" but then also "MTChannel" instead of "MTChannelBroker" ;-)

Reply:
@rhuss I think the current example message is correct. Any objections to proceed and merge this PR?

Reply:
+1 to merge it.